### DISCUSSION TITLE
Synchronize model changes to CRUD applications

### DISCUSSION THREAD

**Discussion by MikeCraig418** [TiagoSilvaPereira/vemto-issues#186] [Ideas & Suggestions]
**Problem:**

I modified one of my models and my CRUD view lost the corresponding fields.

**Proposed Solution:**

Add the ability to sync model changes into all (or selected) CRUDs.

**Comment by TiagoSilvaPereira** [comment]
Hi @949mac generally Vemto tries to sync the changes to the models... could you please record a small video showing what happened?

**Reply by MikeCraig418** [reply]
Thanks for the reply. If I encounter again, I'll thoroughly document.
